### PR TITLE
#347 Specified type to any for errors in catch blocks under the funct…

### PR DESCRIPTION
**Pull request by Ari-S-123**
We specified the error type to any in all four try-catch blocks under the functions package in the backend.

### CODE REVIEW

File: src/backend/functions/change-requests.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ const routes: ApiRoute[] = [
 const handler: Handler = async (event, context) => {
   try {
     return routeMatcher(routes, event, context);
-  } catch (error) {
+  } catch (error: any) {
```
Comment (jamescd18) [inline src/backend/functions/change-requests.ts:52]: I believe there's an `Error` type

**Review by jamescd18 — CHANGES_REQUESTED** [review]
Try not 'any'

File: src/backend/functions/change-requests.ts (hunk 1 above)
Comment (Ari-S-123) [reply src/backend/functions/change-requests.ts:52]: Screenshot (979)
If I try that then this happens. It seems like it has to be either 'any' or 'unknown'.

Comment (KincentLan) [reply src/backend/functions/change-requests.ts:52]: @jamescd18 https://stackoverflow.com/questions/42618089/how-do-you-use-typed-errors-in-async-catch
TLDR; catch clause variables may not have a type annotation

Comment (jamescd18) [reply src/backend/functions/change-requests.ts:52]: Thank you for clarifying! Let's go with just `any` then?
